docs/vc-http-api.yml

```
@@ -143,7 +143,67 @@ components:
           "proofPurpose": "assertionMethod",
           "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..YtqjEYnFENT7fNW-COD0HAACxeuQxPKAmp4nIl8jYAu__6IH2FpSxv81w-l5PvE1og50tS9tH8WyXMlXyo45CA",
         }
-    LinkedDataProofOptions:
+    IssueCredentialOptions:
+      type: object
+      additionalProperties: false
+      description: Options for specifying how the LinkedDataProof is created.
+      properties:
+        verificationMethod:
+          type: string
+          description: The URI of the verificationMethod used for the proof. Default assertionMethod URI.
+        proofPurpose:
+          type: string
+          description: The purpose of the proof. Default: "assertionMethod".
+        created:
+          type: string
+          description: The date and time of the proof (with a maximum accuracy in seconds). Default: current system time.
+        challenge:
+          type: string
+          description: A challenge provided by the requesting party of the proof.
+        domain:
+          type: string
+          description: The intended domain of validity for the proof. For example: website.example
+        credentialStatusMethod:
+          type: string
+          description: The method of credential status to issue the credential including. If omitted credential status will be included.
+      example:
+        {
+          "verificationMethod": "did:example:123#z6MksHh7qHWvybLg5QTPPdG2DgEjjduBDArV9EF9mRiRzMBN",
+          "proofPurpose": "assertionMethod",
+          "created": "2020-04-02T18:48:36Z",
+          "domain": "revocation.example",
+          "challenge": "d436f0c8-fbd9-4e48-bbb2-55fc5d0920a8",
+          "credentialStatusMethod": "RevocationList2020Status",
+        }
+    PresentCredentialOptions:
+      type: object
+      additionalProperties: false
+      description: Options for specifying how the LinkedDataProof is created.
+      properties:
+        verificationMethod:
+          type: string
+          description: The URI of the verificationMethod used for the proof. If omitted a default assertionMethod will be used.
+        proofPurpose:
+          type: string
+          description: The purpose of the proof. If omitted "assertionMethod" will be used.
+        created:
+          type: string
+          description: The date of the proof. If omitted system time will be used.
```

trying to help get @msporny's point across...
```suggestion
        verificationMethod:
          type: string
          description: The URI of the verificationMethod used for the proof. Defaults to ToDo____________ToDo.
        proofPurpose:
          type: string
          description: The purpose of the proof. Defaults to "assertionMethod".
        created:
          type: string
          description: The date of the proof. Defaults to system time.
```

---

note, if the property can be omitted, please mark as `nullable: true` so that a null can correctly be passed in the object or parameter and still pass complete validation checking

---

@mprorock - I think that comment should be posted somewhere with broader readership, than those interested in my change suggestion (which I believe has been obviated by others).

---

good catch - i will likely create an issue on this and a few other specifics related to openapi as best practices

---

Thanks @Tallted suggestions have been adopted in subsequent commits